### PR TITLE
fix touch functionality in Edge

### LINKED ISSUE #49: Touch functionality does not work in Microsoft Edge
I tried using this sample on Microsoft Edge and the touch functionality does not work.
http://ixisio.github.io/bootstrap-touch-carousel/

### PR DESCRIPTION
**Pull request by patrickkettner**
fixes #49

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -27,7 +27,7 @@ module.exports = function(grunt) {
       header: '+function ($) {\n' +
               '  "use strict";\n' +
               '\n' +
-              '  if (!("ontouchstart" in window || navigator.msMaxTouchPoints)) return false \n\n',
+              '  if (!("ontouchstart" in window || "PointerEvent" in window || navigator.msMaxTouchPoints)) return false \n\n',
```
Comment (ixisio) [inline Gruntfile.js:30]: Please use single quotes here `'PointerEvent'`

File: src/js/tests/unit/carousel.js [hunk 2]
```diff
@@ -7,7 +7,7 @@ $(function () {
       // console.log( "ontouchstart" in window ? true: false); // return true in PhantomJS
 
       test("Should run plugin because touchevents are enabled", function() {
-        var hasTouchEvents = ("ontouchstart" in window || navigator.msMaxTouchPoints) ? true : false;
+        var hasTouchEvents = ("ontouchstart" in window || "PointerEvent" in window || navigator.msMaxTouchPoints) ? true : false;
```
Comment (ixisio) [inline src/js/tests/unit/carousel.js]: see above

File: src/js/touch-carousel.js [hunk 3]
```diff
@@ -4,7 +4,7 @@
   /**
    * Return whole plugin if touch is not supported
    */
-  if (!("ontouchstart" in window || navigator.msMaxTouchPoints)) return false;
+  if (!("ontouchstart" in window || "PointerEvent" in window || "navigator.msMaxTouchPoints)) return false;
```
Comment (ixisio) [inline src/js/touch-carousel.js]: see above

File: src/js/transition.js [hunk 4]
```diff
@@ -4,7 +4,7 @@
   /**
    * Return whole plugin if touch is not supported
    */
-  if (!("ontouchstart" in window || navigator.msMaxTouchPoints)) {
+  if (!("ontouchstart" in window || "PointerEvent" in window || navigator.msMaxTouchPoints)) {
```
Comment (ixisio) [inline src/js/transition.js]: see above

**Comment by ixisio** [comment]
Can use also please add an changelog entry for for changes.

**Comment by ixisio** [comment]
hey @patrickkettner 

Thank u very much for changes. I make you some minor comments. Would u please fix it so I can merge it quickly.

Cheers
Andy

File: Gruntfile.js (hunk 1 above)
Comment (patrickkettner) [inline Gruntfile.js:30]: just to be clear, you want to use double quotes on `"ontouchstart"` and single quotes on `\'PointerEvent\'`?

Comment (ixisio) [inline Gruntfile.js:30]: @patrickkettner Oh dam it...didn't realize that! So all my comments below are obsolet. Sorry man :v:

Comment (patrickkettner) [inline Gruntfile.js:30]: no problem! Did you still want a changelog update?

Comment (ixisio) [inline Gruntfile.js:30]: Yeah, that would be nice!

Comment (patrickkettner) [inline Gruntfile.js:30]: updated the changelog as well as the relevant version numbers.

**Pull request closed without merging**